tensorflow/lite/kernels/conv.cc

```diff
@@ -212,6 +258,7 @@ static TfLiteStatus AllocateTemporaryTensorsIfRequired(TfLiteContext* context,
   return kTfLiteOk;
 }
 
+template <KernelType kernel_type>
```

Comment:
This is likely generating a ton of duplicated code.

What you can do is:

TfLiteStatus PrepareImpl(..., KernelType kernel_type) {

}

template<KernelType kernel_type>
TfLiteStatus Prepare() {
  return PrepareImpl(kernel_type);
}

Reply:
@jdduke : I am sorry i could not get completely, how it has generated duplicated code. As per my understanding, the corresponding kernels are passed on to Prepare. Which i can not achieve using the method you suggested.
Please correct me if i am wrong. TIA.

Reply:
We now have multiple template definitions of this entire method, rather than a single definition, because you're using a template. This leads to code bloat.